src/evotorch/neuroevolution/net/runningstat.py

```diff
@@ -1,117 +1,91 @@
-# The following code is adapted from:
-# https://github.com/openai/evolution-strategies-starter/blob/master/es_distributed/es.py.
+# Copyright 2022 NNAISENSE SA
 #
-# The license of OpenAI's original code is as follows:
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
 #
-# The MIT License
+#      http://www.apache.org/licenses/LICENSE-2.0
 #
-# Copyright (c) 2016 OpenAI (http://openai.com)
-#
-# Permission is hereby granted, free of charge, to any person obtaining a copy
-# of this software and associated documentation files (the "Software"), to deal
-# in the Software without restriction, including without limitation the rights
-# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-# copies of the Software, and to permit persons to whom the Software is
-# furnished to do so, subject to the following conditions:
-#
-# The above copyright notice and this permission notice shall be included in
-# all copies or substantial portions of the Software.
-#
-# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
-# THE SOFTWARE.
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
 
-
-from copy import copy, deepcopy
-from typing import Union
+from copy import deepcopy
+from typing import Optional, Union
 
 import numpy as np
+import torch
+from torch import nn
+
+from .runningnorm import RunningNorm
 
 
 class RunningStat:
```

Comment:
As we are modifying RunningStat, would now be a sensible time to introduce decaying running stats?
[It's an option](https://github.com/Denys88/rl_games/blob/master/rl_games/algos_torch/moving_mean_std.py) in the `rl_games` library which is currently used by NVIDIA's [isaacgym](https://github.com/NVIDIA-Omniverse/IsaacGymEnvs) examples.

The feature is low priority, but since we're already modifying the file and the relevant change is small, it could be a nice addition here.

Reply:
Thanks for your comment. Definitely worth considering and implementing in the future. Could you please open an issue? Thanks!

Reply:
@NaturalGradient open issue please